tests/Validator.test.ts

```diff
@@ -0,0 +1,159 @@
+import { resolve } from 'path';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { expect, jest, describe, it, beforeAll, afterAll } from '@jest/globals';
```

Comment:
I cannot get VS Code (through WSL2) to recognize these global jest functions without this line.